File: riftbound-api/delete_card.tmp.py
from sqlalchemy import text
from app.core.database import SessionLocal

def run():
    db = SessionLocal()
    try:
        # Pega o ID do user
        res = db.execute(text("SELECT id, email FROM auth.users WHERE email ILIKE '%ph.sama@gmail%'"))
        row = res.fetchone()
        if not row:
            print("Usuário não encontrado")
            return
            
        user_id = row[0]
        print(f"UID: {user_id}")
        
        # Pega as cartas Ahri
        res = db.execute(text("SELECT id FROM cards WHERE name ILIKE '%Ahri%Alluring%'"))
        card_ids = [row[0] for row in res.fetchall()]
        print(f"Ahri Cards: {card_ids}")
        
        if card_ids:
            # Pega coleções para mostrar antes
            query_print = text("SELECT card_id FROM collections WHERE user_id = :uid AND card_id = ANY(:cid)")
            cols = db.execute(query_print, {"uid": user_id, "cid": card_ids}).fetchall()
            print(f"Possui na Collection: {cols}")
            
            # Deleta as possesões
            query = text("DELETE FROM collections WHERE user_id = :uid AND card_id = ANY(:cid)")
            res = db.execute(query, {"uid": user_id, "cid": card_ids})
            db.commit()
            print(f"Deletadas {res.rowcount} linhas de collections")
            
    except Exception as e:
        print(f"Erro: {e}")
        db.rollback()
    finally:
        db.close()

if __name__ == "__main__":
    run()

File: delete_card.tmp.js
const { Client } = require('pg');

const client = new Client({
  connectionString: 'postgresql://postgres:ujg%21ze%3Fv_%40r2F9Q@db.onvhjecibcjismangnhz.supabase.co:5432/postgres'
});

async function run() {
  try {
    await client.connect();
    // Get user id
    const userRes = await client.query("SELECT id FROM auth.users WHERE email = 'ph.sama@gmail'");
    if (userRes.rowCount === 0) {
      console.log('User not found');
      return;
    }
    const uid = userRes.rows[0].id;
    console.log('User ID:', uid);

    // Get card ids
    const cardsRes = await client.query("SELECT id, name FROM cards WHERE name LIKE '%Ahri%Alluring%'");
    console.log('Cards found:', cardsRes.rows.map(c => c.name));
    
    // Delete from collections
    const deleteRes = await client.query(
      "DELETE FROM collections WHERE user_id = $1 AND card_id IN (SELECT id FROM cards WHERE name LIKE '%Ahri%Alluring%')",
      [uid]
    );
    
    console.log('Rows deleted from collections:', deleteRes.rowCount);
  } catch (err) {
    console.error('Error:', err);
  } finally {
    await client.end();
  }
}

run();
